ansible/hosts

```diff
@@ -1,5 +1,5 @@
 [production]
-svsticky.nl ansible_python_interpreter=/usr/bin/python2.7
+svsticky.nl ansible_python_interpreter=/usr/bin/python3
```

Comment:
Do we need python3 or python3.5?

Reply:
Err, Ansible needs 3.5+, and Ubuntu 18.04 ships with 3.6, so I'll change this to `python3.6`.